crates/shim/src/synchronous/mod.rs

```diff
@@ -111,10 +112,9 @@ pub trait Shim {
     ///
     /// # Arguments
     /// - `runtime_id`: identifier of the container runtime.
-    /// - `id`: identifier of the shim/container, passed in from Containerd.
-    /// - `namespace`: namespace of the shim/container, passed in from Containerd.
+    /// - `args`: command line arguments passed to the shim which includes namespace and id
     /// - `config`: for the shim to pass back configuration information
-    fn new(runtime_id: &str, id: &str, namespace: &str, config: &mut Config) -> Self;
+    fn new(runtime_id: &str, args: &Flags, config: &mut Config) -> Self;
```

Comment:
Could you please update `new()` in `crates/shim/src/asynchronous/mod.rs`

Reply:
I meant to do this, thanks for catching it!